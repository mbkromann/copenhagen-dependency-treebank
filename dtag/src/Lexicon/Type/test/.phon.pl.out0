t: super=[] lvars=[phon=list([af,be,tal,ing]-[], 1)]

