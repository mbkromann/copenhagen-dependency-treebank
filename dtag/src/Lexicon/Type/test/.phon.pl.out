ERROR! cannot read file /opt/dta/arcs.header in Graph->readfile
check that DTAGHOME is set correctly!
ERROR! cannot read file /opt/dta/arcs.trailer in Graph->readfile
check that DTAGHOME is set correctly!
t: super=[] lvars=[phon=list([af,be,tal,ing]-[], 1)]

